localstripe/resources.py

```diff
@@ -2675,6 +2676,7 @@ def __init__(self, customer=None, metadata=None, items=None,
             items = [{'plan': plan, 'quantity': quantity}]
 
         trial_end = try_convert_to_int(trial_end)
+        trial_from_plan = try_convert_to_bool(trial_from_plan)
```

Comment:
Maybe we could add some check on it in the try/except below.
Like:
```python
if trial_from_plan is not None:
     assert type(trial_from_plan) is bool
if trial_end is not None:
     assert not trial_from_plan  # <== it's from the doc
```
For the ` assert not trial_from_plan` it's because [the doc says](https://stripe.com/docs/api/subscriptions/create):
> Setting this flag to true together with trial_end is not allowed.

Reply:
right. added the check